src/main/java/dolla/Ui.java

```diff
@@ -19,7 +19,7 @@
             + "| |_| | |_| |   <  __/\n"
             + "|____/ \\__,_|_|\\_\\___|\n";
 
-    private static String line = "\t____________________________________________________________";
+    public static String line = "\t____________________________________________________________";
```

Comment:
Why the change to public?

Reply:
wanna access it from LimitUi class (might be able to change back to private after moving all of them to the same package AKA dolla.Ui)